index.html

```diff
@@ -22,8 +22,6 @@
 
     <script type="text/javascript"
             src="lib/handlebars-1.0.0.beta.6.js"></script>
-    <script type="text/javascript"
-            src="http://www.google-analytics.com/urchin.js"></script>

```

Comment:
@LuxoftAKutsan why did you remove this?

Reply:
External reference do not allow to run sdlhmi if internet access is limited